Makefile

```diff
@@ -37,7 +37,6 @@ dockertogoarch = $(patsubst arm/v7,arm,$(1))
 
 PLATFORMS ?= linux/amd64,linux/arm64
 IMAGES = submariner-operator submariner-operator-index
-PRELOAD_IMAGES := submariner-operator
```

Comment:
This will now preload the `submariner-operator-index` image as well - is this desired?

Reply:
Its benign and would actually be needed when testing any bundle based deployments in the future.